ride_to_work_by_bike_config.toml

```diff
@@ -59,3 +59,6 @@ defaultDistanceZero = "0.00"
 entryFeePaymentMin = "390"
 entryFeePaymentMax = "2000"
 entryFeePaymentOptions = "500,700"
+
+# API Configuration
+apiBase = "https://dpnk.dopracenakole.cz/rest/"
```

Comment:
For testing purpose API URL should be "https://test.dopracenakole.cz/rest/"

Reply:
Updated.